vaadin-notification-flow-demo/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.vaadin</groupId>
         <artifactId>vaadin-notification-flow-parent</artifactId>
-        <version>2.0-SNAPSHOT</version>
+        <version>2.1-SNAPSHOT</version>
```

Comment:
Nit: I guess we can also bump button below to `2.3.0` on https://github.com/vaadin/vaadin-notification-flow/pull/108/files#diff-dfaf3703db8c8d055f4a4c29e83a4178R59

Reply:
It's a good catch. It may makes sense to remove that webjar dependency altoghether, so I changed it to use the snapshot dependency of button-flow only instead. (Will update the minor when the snapshot is available later).

Reply:
Bumped button snapshot as it is already available.